guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Collecting, storing, managing, and analyzing large quantities of numbers, figures, and files is not a new business activity. However, referring to these numbers, figures, and files as big data is relatively recent. </p><p>The GeoAnalytics Server expands your ArcGIS Enterprise deployment, providing functionality and services to process and analyze big data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Some commas need to be added</p>